frontend_tests/node_tests/emoji_picker.js

```diff
@@ -125,3 +132,61 @@ set_global('popovers', {
     assert($(".emoji-popover-category-tabs").visible());
     assert(!$(".emoji-search-results-container").visible());
 }());
+
+(function test_toggle_emoji_popover() {
+    var element = $('fake-add-emoji');
```

Comment:
@lonerz Actually we can only create things like this 'fake-add-emoji' once. And doing that again would lead to assertion error from zjquery. So what I did is added another commit which enables zjquery to select us a selector if it has already created. This ways for first time use we do need to create this element but next time we just fetch the already created one this way. Though I am not sure of the desired behaviour here. Maybe @showell can clarify expected behaviour here.